Components/Grid/grid.css

```diff
@@ -0,0 +1,23 @@
+@import url(../../style.css);
+@import url(../../css/aside.css);
+
+.grid{
+    display: grid;
+    width: 100%;
+}
+
+.grid-2{
+    grid-template-columns: repeat(2, 1fr);
+    gap: 1rem;
+}
+.grid-3{
```

Comment:
Suggestion: can start `.grid-3{` from line 14 as a line after each class styling block is empty throghout the file.